source/errors.rst

```diff
@@ -90,17 +90,17 @@ duration field, or a project without a name.)
 
   unknown_field_error = {
     "status": 400,
-    "error": "Bad object",
+    "error": "Bad Object",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L112 shows `'Bad object'` as the correct form.